src/main.js

```diff
@@ -1,8 +1,8 @@
 // Create variables targetting the relevant DOM elements here 👇
-var randomCover = document.querySelector('.cover-image')
```

Comment:
Good call changing these to something more readable and that makes more sense in the scope of the project!